danndav/bhankas
# Ft 165491242 user controller api

# What does this PR do?
This PR shows the user controller

Description of Task to be completed?
set up user controller
user signup routes
user login routes
view all users routes
work on LFA feed back
add postgresql to the yml file


# How should this be manually tested?
git fetch and checkout ft-165491242-user-controller-api
checkout to the branch
run the application
navigate to file contoller/usercontroller
click on it to check changes

# Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#165491242

screenshots (if appropriate)
N/A

Questions:
N/A\
